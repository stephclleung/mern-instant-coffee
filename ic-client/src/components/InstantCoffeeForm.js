import React from 'react';

const InstantCoffeeForm = () => {
    return (
        <div>
            <form>
                <input
                    type="text"
                    placeholder="Name of brand, type..."
                    autoFocus
                />
                <input
                    type="number"
                    placeholder="0"
                    min="1"
                    max="500"
                />
                <select>
                    <option defaultValue="package">sticks</option>
                    <option value="container">grams</option>
                </select>
                <input
                    type="number"
                    placeholder="Price"
                    min="0"
                />
                <select>
                    <option defaultValue="CAD">CAD</option>
                    <option value="JPY">JPY</option>
                    <option value="USD">USD</option>
                    <option value="HKD">HKD</option>
                </select>
                <input
                    type="number"
                    placeholder="Acidity"
                    min="1"
                    max="5"
                />
                <input
                    type="number"
                    placeholder="Aroma"
                    min="1"
                    max="5"
                />
            </form>
        </div>
    )
}


export default InstantCoffeeForm;