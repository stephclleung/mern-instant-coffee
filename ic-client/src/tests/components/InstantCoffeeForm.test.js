import React from 'react';
import { shallow } from 'enzyme';
import InstantCoffeeForm from '../../components/InstantCoffeeForm';
import { instantCoffee } from '../fixtures/instant-coffee-data';

test('should render Coffee form correctly', () => {
    const wrapper = shallow(<InstantCoffeeForm/>)
    expect(wrapper).toMatchSnapshot();
})