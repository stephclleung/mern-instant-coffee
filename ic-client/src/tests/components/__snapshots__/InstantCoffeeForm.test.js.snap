// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Coffee form correctly 1`] = `ShallowWrapper {}`;
